examples/face_recognition_11/face_recognition_11/workflow.py

```diff
@@ -0,0 +1,165 @@
+# build out the workflow for the FR problem which is already supported in kolena.fr as a pre-built workflow.
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import dataclasses
+from typing import Optional
+
+from pydantic.dataclasses import dataclass
+
+from kolena.workflow import define_workflow
+from kolena.workflow import EvaluatorConfiguration
+from kolena.workflow import GroundTruth as BaseGroundTruth
+from kolena.workflow import Image
+from kolena.workflow import ImagePair
+from kolena.workflow import Inference as BaseInference
+from kolena.workflow import Metadata
+from kolena.workflow import MetricsTestCase
+from kolena.workflow import MetricsTestSample
+from kolena.workflow.annotation import BoundingBox
+from kolena.workflow.annotation import Keypoints
+
+
+@dataclass(frozen=True)
+class ImageWithMetadata(Image):
+    """
+    An image belonging to an Image Pair containing the locator and metadata
+    for the Face Recognition 1:1 workflow.
+    """
+
+    metadata: Metadata = dataclasses.field(default_factory=dict)
+    """The metadata associated with an image in the image pair."""
+
+
+@dataclass(frozen=True)
+class TestSample(ImagePair):
+    """Test sample type for Face Recognition 1:1 workflow."""
+
+    a: ImageWithMetadata
+    """The locator and metadata associated with image A in the test sample."""
+
+    b: ImageWithMetadata
+    """The locator and metadata associated with image B in the test sample."""
+
+
+@dataclass(frozen=True)
+class GroundTruth(BaseGroundTruth):
+    """Ground truth type for Face Recognition 1:1 workflow."""
+
+    is_same: bool
+    """Whether to treat this image pair as a a genuine pair (True) or an imposter pair (False)."""
+
+
+@dataclass(frozen=True)
+class Inference(BaseInference):
+    """Inference type for Face Recognition 1:1 workflow."""
+
+    left_bbox: Optional[BoundingBox] = None
```

Comment:
Okay, I'll stick with A and B for everything then. It's just because the ImagePair class had A and B that I thought it would be good to be consistent with that.